src/Nowruz/modules/layout/components/linkItem/linkItem.module.scss

```diff
@@ -1,52 +1,50 @@
 @import 'src/components/_primitives.scss';
 .container {
-
-  align-items: flex-start;
-  align-self: stretch;
-  background-color: $color-primary-700;
-  border-radius: 6px;
-  display: flex;
-  gap: 0.5rem;
-  padding: 0.75rem;
-
-  @media screen and (max-width: 600px) {
-    height: 40px;
-    background-color: white;
-  }
+    align-items: flex-start;
+    align-self: stretch;
+    background-color: $color-primary-700;
+    border-radius: 6px;
+    display: flex;
+    gap: 0.5rem;
+    padding: 0.75rem;
+
+    @media screen and (max-width: 768px) {
+        height: 40px;
+        background-color: white;
+    }
 }
 
```

Comment:
I think the color styles have some issues
![image](https://github.com/socious-io/web-app-v2/assets/60564928/cd45d6da-b723-43b5-b1f6-69943f936f62)

Reply:
it's about broken styles and the issue of color is related to the size of the screen.
in this size of screen, we have styles of desktop but it should be the style of mobile and table and the predefined tablet is 768px (md) everywhere